docs/tutorials/lattice-fermions.ipynb

```diff
@@ -0,0 +1,762 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        log_det = jnp.linalg.slogdet(phi)
</pre><p>I would change it to</p><pre class="ql-syntax" spellcheck="false">&nbsp; &nbsp; sign, logabsdet = jnp.linalg.slogdet(phi)
&nbsp; &nbsp; return logabsdet.astype(complex) + jnp.log(sign.astype(complex))
</pre><p>which is more comprehensible (the second conversion to array is not needed)</p><p> </p><p>or even, the one that is slightly more correct because it preserves single-precision if ever used</p><pre class="ql-syntax" spellcheck="false">&nbsp; &nbsp; sign, logabsdet = jnp.linalg.slogdet(phi)
&nbsp; &nbsp; &nbsp; &nbsp; return jnp.log(sign.astype(nk.jax.dtype_complex(phi.dtype))) + logabsdet.astype(nk.jax.dtype_complex(phi.dtype)
</pre><p>but that might be more confusing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/1581/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>